ksqldb-rest-app/src/test/java/io/confluent/ksql/api/integration/ScalablePushBandwidthThrottleIntegrationTest.java

```diff
@@ -280,6 +280,11 @@ private void assertExpectedScalablePushQueries(
             || !metadata.getScalablePushRegistry().get().latestHasAssignment()) {
           return false;
         }
+        try {
```

Comment:
assertThatEventually has a backoff built into it. You can configure it with any of these options:
```
  public static <T> T assertThatEventually(
      final Supplier<String> message,
      final Supplier<? extends T> actualSupplier,
      final Matcher<? super T> expected,
      final long timeout,
      final TimeUnit unit,
      final long initialPausePeriodMs,
      final long maxPausePeriodMs,
      final ExceptionHandling exceptionHandling
  ) {
```

Reply:
added a longer period pause